### ISSUE TITLE
is blue color good for hover ? (link in top menu)

### ISSUE THREAD

**Issue by Pourghannad** [mehrnooshdsht/96-landing#5]
when top menu don't have black background , right menu (<a>) have blue hover .
i think pink background is not good for blue text (link)

**Comment by mehrnooshdsht** [comment]
I agree with you and I prefer shades of grey or something like that. I would be glad if you implement your ideas and we can merge it then :)
